pkg/agent/utils.go

```diff
@@ -586,3 +586,22 @@ func addFeatureGateString(featureGates string, key string, value bool) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// ParseCSEMessage parses the raw CSE output
+func ParseCSEMessage(message string) (*datamodel.CSEStatus, error) {
```

Comment:
the error here should be the one we defined above so consumer can access the code easily, not a general error
type Error struct {